src/utils/dig.js

```diff
@@ -72,12 +72,12 @@ export const handleDig = async ({ domain, types, short, provider }) => {
     };
 
     // Convert results to an embed
-    return results.map(({ type, data }) => createEmbed(`${type} records`, present(type, data), 'diggy diggy hole'));
+    return results.map(({ type, data }) => createEmbed(`${type} records`, present(type, data), 'diggy diggy hole :warning: cd bit set for request, DNSSEC validation disabled'));
 };
 
 export const parseEmbed = embed => {
     // Match the domain name, type and if the short format was requested
-    const descMatch = embed.description.match(/^`(\S+) (\S+) @(\S+) \+noall \+answer( \+short)?`\n/);
+    const descMatch = embed.description.match(/^`(\S+) (\S+) @(\S+) \+noall \+answer( \+short \+cdflag)?`\n/);
```

Comment:
```suggestion
    const descMatch = embed.description.match(/^`(\S+) (\S+) @(\S+) \+noall \+answer( \+short)?( \+cdflag)?`\n/);
```
You need a separate capture group here, otherwise you're never going to detect it being set.